.gitignore

```diff
@@ -21,3 +21,6 @@ earth_enterprise/tutorial/FusionTutorial/KML
 /*.files
 /*.includes
 
+# CLion project files.
+.idea/
```

Comment:
Consider ignoring these directories at all directory levels using the "/**" syntax

Reply:
I also recently read (can't remember where) that because editors are a user preference and editors come and go as preferences change it is a good idea to use `~/.gitignore_global` for those things.  Unless it is a tool the project itself requires the use of.  BTW I really like the `CLion` editor too but I wasn't sure how well it would work without `cmake`.

https://davidwalsh.name/global-gitignore

Reply:
@tst-ccamp good point. I'll add that in.

@tst-rwildes  I have found the Jet brains IDE to be just one of the bests hands down. I was using it for the debugger on this. I was a bit surprised to see that I could not build in a custom build/run functionality. I also find the vim implementation to be great in Clion/Pycharm. As of right now, I found that the most effective way to build/run was just making a small script that I keep in a console window. And then attaching to the process.